app/controllers/projects/creation_wizard_controller.rb

```diff
@@ -54,14 +54,23 @@ def update # rubocop:disable Metrics/AbcSize
       .call(permitted_params.project)
 
     if service_call.success?
-      if params[:finish]
-        redirect_to project_path(@project), notice: I18n.t("projects.wizard.success")
+      if last_page?
+        service_call2 = Projects::CreateArtifactWorkPackageService.new(user: current_user, model: @project).call
+        if service_call2.success?
+          redirect_to project_work_packages_path(@project, @project.project_creation_wizard_artifact_work_package_id),
+                      notice: I18n.t("projects.wizard.success")
+        else
+          flash[:error] = service_call2.errors.full_messages
+          render :show,
+                 locals: { menu_name: :none },
+                 status: :unprocessable_entity
```

Comment:
```suggestion
        creation_call = Projects::CreateArtifactWorkPackageService.new(user: current_user, model: @project).call
        if creation_call.success?
          redirect_to project_work_packages_path(@project, @project.project_creation_wizard_artifact_work_package_id),
                      notice: I18n.t("projects.wizard.success")
        else
          flash[:error] = creation_call.errors.full_messages
          render :show,
                 locals: { menu_name: :none },
                 status: :unprocessable_entity
```

I'd prefer an explicit name over just `service_call2`

Reply:
ah yes, that's a part I wrote quickly initially to get things working. It was not meant to stay like this.

Good that you spotted it.